### DISCUSSION TITLE
Build notes

### DISCUSSION THREAD

**Discussion by electro991** [CrashOverride85/zc95#102] [General]
Here are some notes from my recent build that may be useful to others:

When ordering the output board from JLCPCB, 
the LEDs (D3, D4, D5, D6, D7) part #C72043 were out of stock, i replaced with C2986059
Also, C16 part# C250973 no longer exist on the LCSC site, I replaced with C43348 

On the main board BOM spreadsheet, J16 components could be removed as they are not needed anymore with the current rev of the board.

The placement of the header and orientation for the 2X4P IDC (J19) on the main board is not specified on the silkscreen. I had to make sure I soldered it in the right orientation, cable notch facing the rear of the board. (assuming you create your IDC cable pin to pin)
![image](https://github.com/CrashOverride85/zc95/assets/173577055/5a54c81f-8fd5-40c3-8974-67877a89baa8)

Finally, for the front panel controls board, I have created a 3d printed riser for the 6 LEDs (U3,U4,U5,U6,U7,U8). The LEDs slip (with a little pressure) into it and are automatically the proper height when fitting the front panel. see attached.
[led-riser.zip](https://github.com/user-attachments/files/15944518/led-riser.zip)

Hope this helps

**Comment by CrashOverride85** [comment]
nice, thanks for the the notes, I'll update the BoM with the new part numbers.

Would you mind if I added that photo to the repo & build notes?

Also, any chance of a photo of the led-riser? Sounds like a neat addition :)

**Reply by electro991** [reply]
My pleasure. Feel free to add the photo. 

This is the LED riser. Unfortunately the front panel is now attached but it shows well the LEDs all lined up easily. If too tight depending on printer, can be printed 1% larger.
![image](https://github.com/CrashOverride85/zc95/assets/173577055/20af3fb7-f438-4ed6-b670-86cea62ca377)
